docker-compose.yml

```diff
@@ -107,5 +107,5 @@ services:
       context: ./sdks/js/packages/sdk-demo
     ports:
       - "3000:3000"
-    depends_on:
```

Comment:
why remove the dependency?

Reply:
The network call will fail, UI will still work and API will start working once Frontier instance is Ready.